### PR TITLE
Added glossary term for PIV Credential 

### PR DESCRIPTION
**Pull request by jglosx3**

### CODE REVIEW

File: _Appendix/glossary.md [hunk 1]
```diff
@@ -341,6 +341,10 @@ to an individual that contains a PIV Card application that stores _identity_ _cr
 _cryptographic keys_, digitized fingerprint representation) so that the claimed _identity_ of the _cardholder_ can
 be verified against the stored _credentials_.
 
+PIV Credintial
```
Comment (regenscheid) [inline _Appendix/glossary.md:344]:
```suggestion
PIV Credential
```

File: _Appendix/glossary.md [hunk 2]
```diff
@@ -341,6 +341,10 @@ to an individual that contains a PIV Card application that stores _identity_ _cr
...
+: The credentials that are stored on the PIV card that authoritatively binds
+an identity (and, optionally, additional attributes) to the authenticated card holder.
```
Comment (jricher) [inline _Appendix/glossary.md:346]:
```suggestion
: The _credentials_ that are stored on the _PIV card_ that authoritatively binds
an _identity_ (and, optionally, additional attributes) to the authenticated _cardholder_.
```

**Comment by regenscheid** [comment]
@regenscheid to revise this to make it clear that PIV credentials include both the PKI credentials on PIV cards as well as derived PIV credentials.

**Comment by regenscheid** [comment]
@jglosx3 @hferraio - I revised this to explicitly call out public key certificates on PIV cards as well as derived PIV credentials.

**Review by hferraio — APPROVED** [review]

File: _Appendix/glossary.md [hunk 3]
```diff
@@ -341,6 +341,9 @@ to an individual that contains a PIV Card application that stores _identity_ _cr
 _cryptographic keys_, digitized fingerprint representation) so that the claimed _identity_ of the _cardholder_ can
 be verified against the stored _credentials_.
 
+PIV Credential
+: A _credential_ that authoritatively binds an _identity_ (and, optionally, additional attributes) to the authenticated _cardholder_ that is issued, managed, and used in accordance with the PIV standards. These credentials include _public key certificates stored on a _PIV card_ as well as other _authenticators_ bound to a _PIV account_ as _derived PIV credentials_.
```
Comment (regenscheid) [inline _Appendix/glossary.md:345]:
```suggestion
: A _credential_ that authoritatively binds an _identity_ (and, optionally, additional attributes) to the authenticated _cardholder_ that is issued, managed, and used in accordance with the PIV standards. These credentials include _public key certificates_ stored on a _PIV card_ as well as other _authenticators_ bound to a _PIV account_ as _derived PIV credentials_.
```

Comment (jricher) [inline _Appendix/glossary.md:345]:
```suggestion
: A _credential_ that authoritatively binds an _identity_ (and, optionally, additional attributes) to the authenticated _cardholder_ that is issued, managed, and used in accordance with the PIV standards. These credentials include _public key certificates_ stored on a _PIV card_ as well as other _authenticators_ bound to a _PIV account_ as _derived PIV credentials_.
```

**Pull request merged**